content/security/for-admins/hardened-desktop/settings-management/admin-console-configure.md

```diff
@@ -0,0 +1,68 @@
+---
+description: How to configure Settings Management for Docker Desktop using the Docker Admin Console
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure with the Docker Admin Console
+---
+
+{{% restricted title="Early Access" %}}
+
+Settings Management in Docker Admin Console is in Early Access and is available to Docker Business customers only.
+
+{{% /restricted %}}
+
+This page contains information for administrators on how to configure Settings Management with the Docker Admin Console to specify and lock configuration parameters to create a standardized Docker Desktop environment across your Docker company or organization.
+
+## Prerequisites
+
+- [Download and install Docker Desktop 4.35.0 or later](/desktop/release-notes.md).
+- As an administrator, you need to [enforce
+  sign-in](/security/for-admins/enforce-sign-in/_index.md). This is
+because the Settings Management feature requires a Docker Business
+subscription and therefore your Docker Desktop users must authenticate to your
+organization for configurations to take effect. 
+
+## Create a settings policy
+
+1. Within the [Docker Admin Console](https://admin.docker.com/) navigate to the company or organization you want to define a settings policy for. 
```

Comment:
Maybe add a "Note" that if their orgs are managed under a Company, they will need to manage settings from the Company level. Otherwise they would manage at the organization level